blockchain/citizens/become-congressmen.md

```diff
@@ -77,43 +77,64 @@ To vote, you need to set the correct threshold and proposal. Different actions m
 
 ### Liberland legislation
 
-Congressmen are able to add and remove rights at the international treaty tier. The international treaty tier is represented as "tier=1". Learn more about these methods in `Thresholds`
+Congressmen are able to add and remove legislation at the international treaty tier. The international treaty tier is represented as "tier=1". Learn more about these methods in `Thresholds`.
+
+In Liberland, there are three tiers of international treaties: the "Constitution," "International Treaty," and "Law." These legal frameworks can only be altered by specific groups of individuals.
+
+- Constitution: This tier encompasses the rights outlined in the constitution. Any citizen can propose changes to this law through the "propose_rich_origin" mechanism.
+- International Treaties: These treaties fall under the jurisdiction of the congress. Modifications to these treaties can be made through "motions" initiated by the congress.
+- Law: This tier comprises legislation that can only be modified through the blockchain. Any citizen has the opportunity to submit their own proposals (excluding "propose_rich_origin") to make changes at this level.
+
+### Democracy externel
+
+Whenever the congress votes and approves a proposal, it is implemented using the 'sudo call' (externalPropose, externalProposeMajority, externalProposeDefault). This implies that the proposed action is executed with administrative privileges or rights.
```

Comment:
No, `sudo call` is explicitly a call made by pallet-sudo. If we want to go into details as such, this is a `root-origin call`.